armsrc/util.c

```diff
@@ -410,6 +410,8 @@ uint8_t binaryToHex(char *binaryStr) {
     return (uint8_t)strtoul(binaryStr, NULL, 2);
 }
 
+#define PICOPASS_BLOCK_SIZE 8
```

Comment:
This define already exists in: 
`include/iclass_cmd.h`

Reply:
I just removed the definition and replaced it with an 8 for simplicity.